src/pages/Contributor.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { contributors } from "../Helper"
+import github from "../assets/github.svg"
+import Linkedin from "../assets/Linkedin.svg"
```

Comment:
I would consider a refactor to make `Linkedin` all lowercase. I suggest `linkedin`